x-pack/test_serverless/functional/test_suites/search/inference_management.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ModelConfig } from '@kbn/inference_integration_flyout/types';
+
+import { testHasEmbeddedConsole } from './embedded_console';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects([
+    'svlCommonPage',
+    'svlCommonNavigation',
+    'inferenceManagement',
+  ]);
+  const browser = getService('browser');
+  const ml = getService('ml');
+
+  describe('Serverless Inference Management UI', function () {
+    before(async () => {
+      await pageObjects.svlCommonPage.loginWithRole('admin');
+      await pageObjects.svlCommonNavigation.sidenav.clickLink({
+        deepLinkId: 'searchInferenceEndpoints',
+      });
+    });
+
+    after(async () => {
+      await pageObjects.svlCommonPage.forceLogout();
+    });
+
+    describe('endpoint empty view', () => {
+      it('is loaded successfully', async () => {
+        await pageObjects.inferenceManagement.InferenceEmptyPage.expectComponentsToBeExist();
+      });
+      it('opens add inference flyout', async () => {
+        await pageObjects.inferenceManagement.InferenceEmptyPage.expectFlyoutTobeOpened();
+      });
+    });
+
+    describe('endpoint tabular view', () => {
+      before(async () => {
+        const modelConfig: ModelConfig = {
+          service: 'elser',
+          service_settings: {
+            num_allocations: 1,
+            num_threads: 1,
+          },
+        };
+        await ml.api.createInferenceEndpoint('endpoint-1', 'sparse_embedding', modelConfig);
```

Comment:
Similar to this [comment](https://github.com/elastic/kibana/pull/188438/files#r1683285597), we need to write some clean-up in the after block.